webapp/packages/core-sdk/src/queries/grid/getSqlExecuteTaskResults.gql

```diff
@@ -1,46 +1,47 @@
-mutation getSqlExecuteTaskResults(
-    $taskId: ID!
-) {
-    result: asyncSqlExecuteResults(
-        taskId: $taskId
-    ) {
-        duration
-        statusMessage
-        filterText
-        fullQuery
-        results {
-            title
-            updateRowCount
-            sourceQuery
-            dataFormat
-            resultSet {
-                id
-                columns {
-                    dataKind
-                    entityName
-                    fullTypeName
-                    icon
-                    label
-                    maxLength
-                    name
-                    position
-                    precision
-                    required
-                    readOnly
-                    readOnlyStatus
-                    scale
-                    typeName
-                    supportedOperations {
-                        id
-                        expression
-                        argumentCount
-                    }
-                }
-                rows
-                singleEntity
-                hasMoreData
-                hasRowIdentifier
-            }
+mutation getSqlExecuteTaskResults($taskId: ID!) {
+  result: asyncSqlExecuteResults(taskId: $taskId) {
+    duration
+    statusMessage
+    filterText
+    fullQuery
+    results {
+      title
+      updateRowCount
+      sourceQuery
+      dataFormat
+      resultSet {
+        id
+        columns {
+          dataKind
+          entityName
+          fullTypeName
+          icon
+          label
+          maxLength
+          name
+          position
+          precision
+          required
+          readOnly
+          readOnlyStatus
+          scale
+          typeName
+          supportedOperations {
+            id
+            expression
+            argumentCount
+          }
         }
+        rows
+        rowsWithMetaData {
+          data
+          metaData
+        }
```

Comment:
seems like we need to use [directives](https://graphql.org/learn/queries/#directives) or use only `rowsWithMetaData`